### PR TITLE
Add Command Line Interface Pages project support

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
[Main project page](https://github.com/command-line-interface-pages)

[Here](https://github.com/command-line-interface-pages/syntax/blob/main/base.md) is the current syntax description and a [repo](https://github.com/command-line-interface-pages/cli-pages) with pages.

Render is available [here](https://github.com/command-line-interface-pages/v2-tooling/tree/main/clip-view).

image

Pages for other CLIP project tools will be added later. Accepting this PR will help my project gain more popularity, thanks for understanding.

## Important

This project goal is to replace [TlDr](https://github.com/tldr-pages/tldr) project completely. CLIP project already has better client in terms of feature amount compared to [TlDr official one](https://github.com/tldr-pages/tldr-python-client) and third party clients. To be specific: it [provides](https://github.com/command-line-interface-pages/syntax/blob/main/syntax.md) more expressive and clear syntax for writing pages, [color theme](https://github.com/command-line-interface-pages/prototypes/pull/22) support. Even TlDr maintainers help me to develop it I think their main goal is to make TlDr project better and not mine, so I expect the most of the help from non-TlDr contributors.

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
@srsudar, can u review this PR?

**Comment by srsudar** [comment]
Sorry for the delay, thanks for the ping. A few points:

1. Can you add two spaces after each example? Check out the `find` example for the syntax I copy:

https://raw.githubusercontent.com/srsudar/eg/master/eg/examples/find.md

I outline the requirements I'm working with here. Not sure I'd choose these again, but I'm sticking with them for now:

https://github.com/srsudar/eg#formatting

2. Can you also add a `# Basic Usage` section like is in `find`? That is something I'd like to keep across all pages.

3. The examples in the first section you should be able to run directly. These are all examples in the first section, and I don't think you can run them directly? Is the `...` something you pass directly to the shell? I don't think it is but I'm not positive...

Something like this:

```
render specific local pages

    clip-view path/to/page1.clip path/to/page2.clip ..
```

should be more like:

```
render the local pages page1.clip and page2.clip

    clip-view path/to/page1.clip path/to/page2.clip
```

Same here, this shouldn't show the available options, this should choose one:

```
render page_name1 and page_name2 using the tldr-colorful renderer

    clip-view --render tldr-colorful page_name1 page_name2
```

All the examples in the first section should follow that syntax.

**Comment by EmilySeville7cfg** [comment]
Thanks, I will fix these issues soon. 😁

**Comment by EmilySeville7cfg** [comment]
@srsudar, problems fixed. 😄

**Comment by EmilySeville7cfg** [comment]
@srsudar, any updates?

File: eg/examples/clip-view.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# clip-view
+
+# Basic usage
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply eg/examples/clip-view.md]: Done.

**Comment by srsudar** [comment]
Nice, added a couple new lines and merged. Will go out in the next version.

**Comment by EmilySeville7cfg** [comment]
Thanks! ☺️

**Pull request merged**